eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriterTest.java

```diff
@@ -156,6 +156,39 @@ public void saveTwoAccession() throws Exception {
         }
     }
 
+    @Test
+    @DirtiesContext
+    public void variantInsertionCheckOrder() throws Exception {
+        SubmittedVariant firstVariant = new SubmittedVariant("assembly", TAXONOMY, "project", CONTIG_1, 2,
+                                                             "C", "A", false);
+        SubmittedVariant secondVariant = new SubmittedVariant("assembly", TAXONOMY, "project", CONTIG_1, 2,
+                                                              "", "A", false);
+
+        accessionWriter.write(Arrays.asList(firstVariant, secondVariant));
+
+        List<AccessionWrapper<ISubmittedVariant, String, Long>> accessions =
+                service.get(Arrays.asList(firstVariant, secondVariant));
+        assertEquals(2, accessions.size());
+
+        int firstVariantLineNumber = getVariantLineNumberByPosition(output, CONTIG_1 + "\t" + "2");
+        //secondVariant position is 1 because it is an insertion and the context base is added
+        int secondVariantLineNumber = getVariantLineNumberByPosition(output, CONTIG_1 + "\t" + "1");
+        assertTrue(firstVariantLineNumber > secondVariantLineNumber);
+    }
+
+    private static int getVariantLineNumberByPosition(File output, String position) throws IOException {
+        BufferedReader fileInputStream = new BufferedReader(new InputStreamReader(new FileInputStream(output)));
+        String line;
+        int lineNumber = 0;
+        while ((line = fileInputStream.readLine()) != null) {
+            if (line.startsWith(position)) {
+                return lineNumber;
+            }
+            lineNumber++;
+        }
+        throw new IllegalStateException("VCF didn't have any data lines");
```

Comment:
The same comment that in this method in the other test:

This message is not accurate. If the VCF has data lines, but the "position" we are looking is not in the VCF, the exception will be thrown as well. Maybe this message is better:
"The VCF does not contain any variant with position " + position